### PR TITLE
Directory

### PR DESCRIPTION
**Pull request by cnguyen0**
Completed the directory's page! Please take a look!

### CODE REVIEW

File: src/PageContent/Directory/DirectoryCard.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import "typeface-roboto";
```
Comment (huangjoyce3) [inline src/PageContent/Directory/DirectoryCard.js]: Change to Open Sans. In my PR #6 I added `<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">` into index.html so you shouldn't have a problem rendering that font once my pr is merged.

File: src/PageContent/Directory/DirectoryCard.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+// Component for directory card
+
+class DirectoryCard extends Component {
+  render() {
+    // this.props.accountType
+		return (
+			<div className='card'>
+				<h1 className='orgName'>{ this.props.organization }</h1>
```
Comment (huangjoyce3) [inline src/PageContent/Directory/DirectoryCard.js]: tab space looks too large (is it 8 spaces?) It should be 4 spaces.

File: src/PageContent/Directory/DirectoryCard.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+						<h3><b>Hours of Operation</b></h3>
+						<p className='notes'>M-F: 9AM-6PM</p>
+						<p>Sat: 10-4pm</p>
+						<p>Sun: Closed</p>
+					</div>
+					<div className='card-detail-3'>
+						<h3><b>Street Address</b></h3>
+						<p className='note'>123 Seasame St.</p>
+						<p className='note'>Seattle, WA 98115</p>
+					</div>*/}
```
Comment (huangjoyce3) [inline src/PageContent/Directory/DirectoryCard.js]: Is this comment intended? What do you mean second row of card details?

File: src/PageContent/Directory/DirectoryFilter.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './Directory.css';
+import "typeface-roboto";
+
+// Component for directory filter
+
+class DirectoryFilter extends Component {
+  render() {
```
Comment (huangjoyce3) [inline src/PageContent/Directory/DirectoryFilter.js]: line 8: 2 spaces, line 10: fix tabs

File: src/PageContent/Directory/DirectoryPage.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+import './Directory.css';
+import "typeface-roboto";
+
+// Component for directory card
+
+class DirectoryPage extends Component {
+
+    render() {
+		return (
+            <div className="directoryPage">
```
Comment (huangjoyce3) [inline src/PageContent/Directory/DirectoryPage.js]: nity grity but line 13 is not indented properly under return

File: src/PageContent/Directory/DirectoryPage.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import DirectoryCard from './DirectoryCard.js';
+import DirectoryFilter from './DirectoryFilter.js';
+import './Directory.css';
+import "typeface-roboto";
```
Comment (huangjoyce3) [inline src/PageContent/Directory/DirectoryPage.js]: open sans

File: src/PageContent/Directory/Directory.css [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+.number {
+    font-size: 14px;
+}
+
+.card, .filter {
+    margin: 20px;
+    display: block;
+    overflow: hidden;
+    font-family: Roboto;
```
Comment (huangjoyce3) [inline src/PageContent/Directory/Directory.css]: open sans

File: src/PageContent/Directory/DirectoryCard.js [hunk 8]
```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import "typeface-roboto";
+
+// Component for directory card
+
+class DirectoryCard extends Component {
+  render() {
+    // this.props.accountType
```
Comment (huangjoyce3) [inline src/PageContent/Directory/DirectoryCard.js]: delete if not needed

File: src/PageContent/Directory/DirectoryFilter.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './Directory.css';
+import "typeface-roboto";
```
Comment (huangjoyce3) [inline src/PageContent/Directory/DirectoryFilter.js]: delete

File: src/PageContent/Directory/DirectoryPage.js (hunk 5 above)
Comment (arielin3) [reply src/PageContent/Directory/DirectoryPage.js]: return itself isn't indented correctly either

File: src/PageContent/Directory/DirectoryPage.js [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+class DirectoryPage extends Component {
+
+    render() {
+		return (
+            <div className="directoryPage">
+                {/* TODO: For now, I'll keep the filter component separate
+                            but I think I will move it to the Directory Page so that it will be easier
+                            to grab the filter query and use it to display the directory cards below */}
+                <DirectoryFilter />
```
Comment (arielin3) [inline src/PageContent/Directory/DirectoryPage.js]: 1. In the constructor, create a `receivingAgencies` and a `delivererGroups` lists in `state` that are filled with dummy data (`id` and `name` data). Just like [here](https://github.com/huangjoyce3/uw-foodwaste/blob/979872d9d066ce5f53ff6d473befa6fa21d73e6c/src/PageContent/RequestPickup/RecurringPickupRequest.js#L15-L22) in Joyce's branch.
2. Pass the list down to `DirectoryFilter` and populate the dropdown list like [here](https://github.com/huangjoyce3/uw-foodwaste/blob/979872d9d066ce5f53ff6d473befa6fa21d73e6c/src/PageContent/RequestPickup/RecurringPickupRequest.js#L222-L230).

**Comment by arielin3** [comment]
taken over by @leontaolong in #32 
Closing it.

**Pull request closed without merging**